nexus/tests/integration_tests/unauthorized_coverage.rs

```diff
@@ -138,7 +138,20 @@ fn test_unauthorized_coverage() {
     let expected_uncovered_endpoints =
         std::fs::read_to_string("tests/output/uncovered-authz-endpoints.txt")
             .expect("failed to load file of allowed uncovered endpoints");
-    assert_eq!(expected_uncovered_endpoints, uncovered_endpoints);
+    let mut unexpected_uncovered_endpoints = "These endpoints were expected to be covered by the unauthorized_coverage test but were not:\n".to_string();
```

Comment:
It seems like we're no longer checking whether there are things in the allowlist file that _are_ covered.  That seems bad?  It feels like it undermines the point of an allowlist.  (why wouldn't we just put every endpoint into the allowlist?)

Reply:
This is bending my brain a little. So, this would happen when you've set something in the allowlist that you expect to _not_ be covered by tests but the tests cover it anyway. So the `uncovered_endpoints` would be _smaller_ than the `expected_uncovered_endpoints` meaning you might need to remove something from the allowlist. 

That's fair. I think the thing I was trying to solve here is there failure experience is _really bad_. I've hit up against this test a lot and it always takes me a while to figure out what went wrong and what I need to do. Ultimately my goal here is just to make it more explicit as to what needs to be done. I'll see if I can work backwards and test both ways without making it messier.

Reply:
That sounds good.  Yeah, I'm all for making this easier.  We could also use expectorate here like we do in other places.  I think the suggestion _not_ to was based on not making it seem like the right thing for people to _add_ things to this list.